### PR TITLE
Feature/usable dev user

### PR DESCRIPTION
**Pull request by schmijos**
Together with the docker configuration it's a bit cumbersome to reregister on restarting the dev server.

Merge https://github.com/noproblan/npl-website/pull/71 first

### CODE REVIEW

File: db/seeds.sql [hunk 1]
```diff
@@ -21,7 +21,8 @@ SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
 --
 INSERT INTO `npl_users` (`id`, `username`, `password`, `salt`, `mail`, `active`, `mail_verified`, `register_datetime`, `written_datetime`) VALUES
 (1, 'admin', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'admin@noproblan.ch', 1, 1, '2012-01-01 00:00:00', '2012-01-01 00:00:00'),
-(2, 'guest', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'noreply@noproblan.ch', 1, 1, '2012-01-01 00:00:00', '2012-01-01 00:00:00');
+(2, 'guest', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'noreply@noproblan.ch', 1, 1, '2012-01-01 00:00:00', '2012-01-01 00:00:00'),
+(3, 'sadi', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'sadi@noproblan.ch', 1, 1, '2017-01-01 00:00:00', '2017-01-01 00:00:00');
```
Comment (schmijos) [inline db/seeds.sql]: Is that enough? Or do I need to give this user some route rights?

File: db/seeds.sql [hunk 2]
```diff
@@ -33,7 +34,10 @@ INSERT INTO `npl_users` (`id`, `username`, `password`, `salt`, `mail`, `active`,
 
 INSERT INTO `npl_userroles` (`id`, `user_id`, `role_id`, `written_datetime`) VALUES
 (1, 1, 1, '2012-01-01 00:00:00'),
-(2, 2, 2, '2012-01-01 00:00:00');
+(2, 2, 2, '2012-01-01 00:00:00'),
+(3, 3, 2, '2017-01-01 00:00:00'),
+(4, 3, 3, '2017-01-01 00:00:00'),
+(5, 3, 5, '2017-01-01 00:00:00');
```
Comment (schmijos) [inline db/seeds.sql]: I think there's something wrong with the resource groups here. The live db hopefully looks different.

Comment (Chuvisco88) [inline db/seeds.sql]: What do you mean "wrong"? As I remember it should be a normal ACL where you provide roles to users, nothing too fancy

File: db/seeds.sql (hunk 1 above)
Comment (Chuvisco88) [inline db/seeds.sql]: The user needs to be added and he needs to get the correct group as well (group id should be 3 for a gamer)

**Comment by schmijos** [comment]
Has been solved with a `dev-user`.

**Pull request closed without merging**